readme

```diff
@@ -0,0 +1,3 @@
+I Changed the link to the wiki so that it opened in a new window instead of the same. I learned how to use github to pull and push code and it was extremely helpful to be able to practice with a real organisation. 
```

Comment:
Hey @Hurricrane! This is not required to be committed. You can add this in the description of the pull request. :)